### PR TITLE
FR-tags

### PR DESCRIPTION
**Pull request by iviondayjr**
Added logic for product tag in FR. Note: Resubmit form required to see changes in text output (tags, product name, etc)

### CODE REVIEW

File: client/src/components/App.jsx [hunk 1]
```diff
@@ -99,17 +99,33 @@ class App extends React.Component {
           price = data.listPrice.split('.')[0];
         }
         const tags = [];
-        if (data.isNew) {
-          tags.push('NEW');
+        if (country == 'caFR') {
```
Comment (rvrita) [inline client/src/components/App.jsx]: Strict equality (===) is safer to use (prevent type coercion: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Strict_equality)

File: client/src/components/App.jsx [hunk 2]
```diff
@@ -99,17 +99,33 @@ class App extends React.Component {
...
         }
-        if (data.isSephoraExclusive) {
-          tags.push('ONLY<span>&nbsp;</span>AT<span>&nbsp;</span>SEPHORA');
-        }
-        if (data.isLimitedEdition) {
-          tags.push('LIMITED<span>&nbsp;</span>EDITION');
-        }
-        if (data.isOnlineOnly) {
-          tags.push('ONLINE<span>&nbsp;</span>ONLY');
+        else {
```
Comment (rvrita) [inline client/src/components/App.jsx]: This is just a tiny formatting issue, else should be in the same line as }.
```} else {```

**Review by rvrita — APPROVED** [review]
Looks great, 2 small fixes but after go ahead and merge!

**Comment by choiwa** [comment]
Looks good to me!

**Pull request merged**